dags/error_aggregates.py

```diff
@@ -1,29 +1,39 @@
-import datetime
+# Generated via query_scheduling/generate_airflow_dags
```

Comment:
We should make sure to update the DAG generation so that this line specifies bigquery-etl. We might want to use the full GH URL: `Generated via https://github.com/mozilla/bigquery-etl/blob/master/bigquery_etl/query_scheduling/generate_airflow_dags.py`